### PR TITLE
Future/navigation

### PR DESCRIPTION
**Pull request by KarolGin**

### CODE REVIEW

File: src/components/Nav/HamburgerMenu/HamburgerMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import "./HamburgerMenu.scss";
+
+export const HamburgerMenu = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const toggleMenu = () => {
+    setIsOpen(!isOpen);
```
Comment (szu-szy) [inline src/components/Nav/HamburgerMenu/HamburgerMenu.tsx:7]: please use prev state to this operation

File: src/components/Nav/HamburgerMenu/HamburgerMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  };
+  return (
+    <div>
+      <button className="hamburger" onClick={toggleMenu}>
+        &#9776;
+      </button>
+      <ul className={`menu ${isOpen ? "open" : ""}`}>
+        <li>
+          <a href="/">Strona Główna</a>
```
Comment (szu-szy) [inline src/components/Nav/HamburgerMenu/HamburgerMenu.tsx:16]: please create object with paths to reuse paths in other files

File: src/pages/EditBooks/EditBooks.tsx [hunk 3]
```diff
@@ -60,6 +61,9 @@ export const EditBooks = () => {
 
     return (
         <>
+        <div className="hamburger-menu-container">
```
Comment (szu-szy) [inline src/pages/EditBooks/EditBooks.tsx:64]: if you created HamburgerMenu you can move this div into this component

Comment (KarolGin) [reply src/pages/EditBooks/EditBooks.tsx:64]: Done

**Review by szu-szy — APPROVED** [review]

**Pull request merged**